### ISSUE TITLE
Make Jet search WCAG v2 compatible.

### ISSUE THREAD

**Issue by erotser** [CrocoBlock/suggestions#7012]
It's a very easy fix, but currently jet search will give you a WCAG error for missing label.

the change is simple: 
the file form.php in /wp-content/plugins/jet-search/templates/jet-ajax-search/global/

needs to change 
<input class="jet-ajax-search__field" type="search" placeholder="<?php echo esc_attr( $settings['search_placeholder_text'] ); ?>" value="" name="s" autocomplete="off" />

into this: 

<input class="jet-ajax-search__field" type="search" aria-label="Search" title="Search" id="search-input" placeholder="<?php echo esc_attr( $settings['search_placeholder_text'] ); ?>" value="" name="s" autocomplete="off" />

**Comment by Crocoblock** [comment]
Released in version 3.2.1
